pkg/controller/machine/machine_controller.go

```diff
@@ -40,6 +47,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const (
+	// ExcludeNodeDrainingAnnotation annotation explicitly skips node draining if set
+	ExcludeNodeDrainingAnnotation = "machine.cluster.sigs.k8s.io/exclude-node-draining"
```

Comment:
Should we move this one into machine_types.go?

Reply:
Seems sensible.